infer/src/bufferoverrun/bufferOverrunDomainRelation.ml

```diff
@@ -6,7 +6,6 @@
  *)
 
 open! IStd
-open Core
```

Comment:
Why no `Core_kernel` here?

Reply:
The entire `open` was unused (as it is implicitly done by `open IStd`), so I removed it.